src/components/block/block.tsx

```diff
@@ -107,7 +107,11 @@ export class Block implements ConditionalSlotComponent, InteractiveComponent {
   /**Block description */
   @Prop() description: string;
 
-  /** When true, removes padding for the slotted content */
+  /**
+   * When true, removes padding for the slotted content
+   *
+   * @deprecated Use `--calcite-block-padding`  CSS variable instead.
```

Comment:
Looks like an extra space before `  CSS`.

@geospatialem do we refer to CSS variables as `variables` or `custom properties`? 

Custom properties (--*): CSS variables

Reply:
We'd prefer `variables` for consistency, also perhaps better to keep separated from our custom element properties.